.azure-pipelines/common/build.yml

```diff
@@ -1,8 +1,8 @@
 steps:
 - task: NodeTool@0
-  displayName: 'Use Node 16.x'
+  displayName: 'Use Node 18.x'
   inputs:
-    versionSpec: 16.x
+    versionSpec: 18.x
```

Comment:
An alternative that puts version into exactly one place (.nvmrc):
steps:
```yaml
  - task: NodeTool@0
    displayName: Using Node.js
    inputs:
      versionSource: fromFile
      versionFilePath: .nvmrc
```

Reply:
Oh nice. Is that new? I thought I looked for this option when I added the .nvmrc files

Reply:
I'm not sure